packages/@tailwindcss-vite/src/index.ts

```diff
@@ -87,28 +87,28 @@ export default function tailwindcss(): Plugin[] {
       },
     })
 
-    let result = scanDir({
+    scanDirResult = scanDir({
       // TODO: This might not be necessary if we enable/disabled auto content
-      // detection
-      base: inputBasePath, // Root directory, mainly used for auto content detection
-      contentPaths: globs.map((glob) => ({
+      // detection.
+      base: config!.root, // Root directory, mainly used for auto content detection
```

Comment:
Maybe we can for now just disable auto-content when base is set to an empty string? We only need that API for this caller anyways.

Reply:
Good idea, instead of an empty string I made it an `Option<…>` in Rust and just removed it from here.

This will now disable auto content detection.